tests/integration/template_weekly_bundles.cfg

```diff
@@ -95,11 +95,12 @@ years = "1985:1989:2",
   mapping_file = ""
   vars = "RIVER_DISCHARGE_OVER_LAND_LIQ"
 
-[tc_analysis]
-active = True
-bundle = "bundle3" # Let bundle1 finish first because "e3sm_diags: atm_monthly_180x360_aave_mvm" requires "ts: atm_monthly_180x360_aave"
-scratch = "#expand scratch#zppy_weekly_bundles_scratch/#expand unique_id#/#expand case_name#"
-years = "1985:1989:2",
+# TODO: Add "tc_analysis" back in after empty dat is resolved.
```

Comment:
`tc_analysis` must be excluded for now because of https://github.com/E3SM-Project/e3sm_diags/pull/866#issuecomment-2405873119